backend/utils/stt/deepgram_util.py

```diff
@@ -219,3 +221,11 @@ def connect_to_deepgram(on_message, on_error, language: str, sample_rate: int, c
         return dg_connection
     except Exception as e:
         raise Exception(f'Could not open socket: {e}')
+
+def convert_pcm8_to_pcm16(data):
```

Comment:
@0xzre I think Joan is correct. If you convert it from 8-bit to 16-bit it will cause the audio to be garbled. This is all that I had to do to deal with opus vs pcm8:

```
data = await self.websocket.receive_bytes()

if self.codec == "opus":
    frame_size = 160
    data = self.opus_decoder.decode(data, frame_size)
                
buffer.extend(data)
```

Reply:
@josancamon19 Yup I mistook the 8 with pcm 8-bit encoding out there, gonna remove that processing now, thanks
@johnmccombs1 Yeah I recorded what the server sent to DG and that's garbled but DG able to translate it then I wonder why :D, turns out its 8khz, thanks